api/src/Controller/CouponKeyController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller;
+
+use App\Service\CampCouponService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class CouponKeyController extends AbstractController {
+    public function __construct(
+        private string $appEnv,
+        private CampCouponService $couponService
+    ) {
+    }
+
+    #[Route('/coupon', name: 'coupon')]
+    public function createAction() {
+        // If not Dev-Environment; Auth as Admin is required
+        if ('dev' !== $this->appEnv) {
```

Comment:
I'd avoid making this dependent on the environment. If we need coupons in dev environment, it's easy to log in with an admin account.